andela/kiba-cfh
# Feature/144937453/users should receive a jwt on successful signup and login

#### What does this PR do?
This pull request assigns a JWT token to users on successful sign in and signup to the site.

#### Description of Task to be completed?
* Users should receive a JWT on successful signup
* Users should receive a JWT on successful login

#### How should this be manually tested?
* It can be tested using Postman.
* After successful signup or login, check cookies for the JWT.
* Unauthorised users cannot have access to the protected /api route.

#### Any background context you want to provide?
JWT is used majorly for authentication, once a user is logged in and assigned a JWT, each subsequent request will include the JWT assigned to the user.

#### What are the relevant pivotal tracker stories?
Users should receive a JWT on successful login #144937451
Users should receive a JWT on successful signup #144937453

#### Screenshots (if appropriate)
<img width="964" alt="screen shot 2017-05-25 at 11 24 32 am" src="https://cloud.githubusercontent.com/assets/28450177/26446743/4df9e9b2-413d-11e7-9897-510e35e27fb9.png">



#### Questions:
N/A
